.github/workflows/test-backend.yml

```
@@ -66,11 +66,10 @@ jobs:
       - name: Install dependencies
         run: docker compose --project-name $STACK_NAME run --rm backend uv pip install -e .
       - name: Migrate DB
-        run: docker compose --project-name $STACK_NAME run --rm -e TESTING=true -e TEST_MODE=true backend uv run bash scripts/prestart.sh
+        run: docker compose --project-name $STACK_NAME run --rm backend uv run bash scripts/prestart.sh
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:1:31: Double quote to prevent globbing and word splitting

(shellcheck)